docs/dev/libvirt/README.md

```diff
@@ -16,13 +17,14 @@ Before you begin, install the [build dependencies](../dependencies.md).
 Make sure you have KVM enabled by checking for the device:
 
 ```console
-$ ls -l /dev/kvm 
+$ ls -l /dev/kvm
```

Comment:
Having the whitespace/formatting changes in their own commit would have been nice :)

Reply:
Done